user_manual/docs/firmware.md

```diff
@@ -5,18 +5,39 @@ Updating Firmware
 
 # Overview {#update-overview}
 
-Discuss boot loader vs application firmware.
+The **Firmware** is the code that runs on the board.
+
+The **Bootloader** is a mode that allows the firmware to be written to the board
 
 # Application Firmware Update {#update-steps}
 
 ## Enter Bootloader Mode {#update-bootloader}
+To enter the Bootloader mode hold the button closest to the usb port when powering up. If you are successful the top LED will be solid and the bottom three LEDS will flash together
```

Comment:
Just a note, we should probably target the instructions to the Number1, since that's what people will be using. In the case of the Number1 there is only 1 button.